src/google/cloud/ndb/query.py

```diff
@@ -914,7 +950,7 @@ class Query:
         filters (Union[Node, tuple]): Node representing a filter expression
             tree. Property filters applied by this query. The sequence
             is ``(property_name, operator, value)``.
-        orders (Union[QueryOrder, list]): The field names used to
+        order_by (Union[tuple, list]): The field names used to
```

Comment:
Let's use `order` to be consistent with `datastore`.

Reply:
The problem is that all of these parameters become query attributes, so order would clash with the order method defined below. That's why I used order_by.

Reply:
@chrisrossi Let me know if you have any thoughts on this. Thanks.

Reply:
I commented (or so I thought) but my comment seems to have disappeared or not gone through, sorry. This is fine. Thank you for the explanation.